src/SquaresOfSortedArray.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Arrays;
+
+public class SquaresOfSortedArray {
+
+    public static void main(String[] args) {
+        System.out.println(Arrays.toString(sortedSquares(new int[]{-4, -1, 0, 3, 10})));
+    }
+
+    public static int[] sortedSquares(int[] nums) {
+        for (int i = 0; i < nums.length; i++) {
+            nums[i] = nums[i] * nums[i];
+        }
+        for (int i = 0; i < nums.length - 1; i++) {
```

Comment:
This is sorting array with time complexity of O(n)2 which is very bad. You are using a nested array. Instead of that use Arrays.sort method which use quick sort, and have time complexity of OnLog(n) which is better than O(n)2